content/ja/docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -30,7 +30,7 @@ StatefulSetの通常の操作では、StatefulSet Podを強制的に削除する
 kubectl delete pods <pod>
 ```
 
-上記がグレースフルターミネーションにつながるためには、`pod.Spec.TerminationGracePeriodSeconds`に0を指定しては**いけません**。`pod.Spec.TerminationGracePeriodSeconds`を0秒に設定することは安全ではなく、StatefulSet Podには強くお勧めできません。グレースフル削除は安全で、kubeletがapiserverから名前を削除する前に[Podが適切にシャットダウンする](/docs/user-guide/pods/#termination-of-pods)ことを保証します。
+上記がグレースフルターミネーションにつながるためには、`pod.Spec.TerminationGracePeriodSeconds`に0を指定しては**いけません**。`pod.Spec.TerminationGracePeriodSeconds`を0秒に設定することは安全ではなく、StatefulSet Podには強くお勧めできません。グレースフル削除は安全で、kubeletがapiserverから名前を削除する前に[Podが適切にシャットダウンする](/ja/docs/concepts/workloads/pods/pod/#termination-of-pods)ことを保証します。
```

Comment:
You could also link to `/ja/docs/concepts/workloads/pods/pod/#podの終了` and use the original fragment identifier.

Reply:
If it is left the original line, the link to here need to contains Japanese characters, which will encoded and hard to read in browser's address bar.

Reply:
Ah, right. I think modern browsers (I use Firefox and Chrome) can decode fragment identifiers though.

You can see this yourself; try visiting [Kubernetesコントロールプレーン](https://kubernetes.io/ja/docs/concepts/#kubernetes%e3%82%b3%e3%83%b3%e3%83%88%e3%83%ad%e3%83%bc%e3%83%ab%e3%83%97%e3%83%ac%e3%83%bc%e3%83%b3) and looking at the address bar.
Of course, each browsers can behave differently.

Reply:
@sftim yes, modern browsers can decode it, but if a user copy the URL from address bar, the clipboard will be `https://kubernetes.io/ja/docs/concepts/#kubernetes%e3%82%b3%e3%83%b3%e3%83%88%e3%83%ad%e3%83%bc%e3%83%ab%e3%83%97%e3%83%ac%e3%83%bc%e3%83%b3`, which is very hard to understand what it is.

Reply:
@sftim Please understand this is fairly necessary. You can see how it is if you try pasting these links on Slack or something.

Reply:
It sounds like this might be a good suggestion to make in https://kubernetes.io/docs/contribute/localization/ as well. What do you think @inductor?

To be clear, I'm talking about making a _suggestion_ and not a _requirement_.

Reply:
@sftim, this is only our local rule, but I think we can ask for other 2byte letters language people like Chinese/Korean/Russian?

Reply:
Please would you?

Reply:
@sftim As Japanese localization team, we are not forcing this to be an official rule. So we decided not to take any action this time.